lib/sidekiq_simple_delay/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module SidekiqSimpleDelay
-  VERSION = '0.2.1'
+  VERSION = '0.2.2'
```

Comment:
Typically, we leave the version alone in the PR, then we bump it master after the PR has been merged. The CHANGELOG can have a HEAD or current section at the top that gets turned into the next version